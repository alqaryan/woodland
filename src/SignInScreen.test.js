import renderer from 'react-test-renderer';
import React from 'react';

const SignInScreen = require('./SignInScreen.js');
describe('SignInScreen', function() {
    it('should be exist', function() {
      expect('SignInScreen').toBeDefined();
    });

    it('should be a function', function(){
      expect(typeof SignInScreen).toBe("function");
    })

    it('renders correctly', () => {
      const tree = renderer.create(<SignInScreen/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });