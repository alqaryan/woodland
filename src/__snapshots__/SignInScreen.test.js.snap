// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInScreen renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 18,
        "fontWeight": "400",
        "marginTop": 32,
        "textAlign": "center",
      }
    }
  >
    Welcome to Woodland!
 Sign In to discover
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 72,
        "justifyContent": "center",
        "marginHorizontal": 30,
      }
    }
  >
    <Text />
  </View>
  <View
    style={
      Object {
        "marginBottom": 48,
        "marginHorizontal": 30,
      }
    }
  >
    <View>
      <Text
        style={
          Object {
            "color": "#8A8F9E",
            "fontSize": 10,
            "textTransform": "uppercase",
          }
        }
      >
        Email Address
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCapitalize="none"
        onChangeText={[Function]}
        rejectResponderTermination={true}
        style={
          Object {
            "borderBottomColor": "#8A8F9E",
            "borderBottomWidth": 0.5,
            "color": "#8A8F9E",
            "fontSize": 15,
            "height": 40,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <View
      style={
        Object {
          "marginTop": 32,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#8A8F9E",
            "fontSize": 10,
            "textTransform": "uppercase",
          }
        }
      >
        Password
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCapitalize="none"
        onChangeText={[Function]}
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "borderBottomColor": "#8A8F9E",
            "borderBottomWidth": 0.5,
            "color": "#8A8F9E",
            "fontSize": 15,
            "height": 40,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
  </View>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
    style={
      Array [
        Object {
          "overflow": "hidden",
        },
        undefined,
      ]
    }
  >
    <View
      accessible={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#80bfb7",
          "borderRadius": 4,
          "height": 52,
          "justifyContent": "center",
          "marginHorizontal": 30,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFF",
            "fontWeight": "500",
          }
        }
      >
        Sign in
      </Text>
    </View>
  </RNGestureHandlerButton>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
    style={
      Array [
        Object {
          "overflow": "hidden",
        },
        undefined,
      ]
    }
  >
    <View
      accessible={true}
      style={
        Object {
          "alignSelf": "center",
          "marginTop": 15,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#414959",
            "fontSize": 15,
          }
        }
      >
        New Explorer? 
        <Text
          style={
            Object {
              "color": "orange",
              "fontWeight": "500",
            }
          }
        >
          Sign up
        </Text>
      </Text>
    </View>
  </RNGestureHandlerButton>
</View>
`;
